src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* max-width: 1200px;
+  min-width: 1024px; */
+  padding: 0 40px 0 50px;
+  margin: 0 auto;
+  background-color: #fff;
+  box-shadow: 0 2px 4px #e2e7ea;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav_item {
+  margin: 0 10px;
```

```suggestion
  margin-right: 20px;
```
Якщо придивитись до дизайну, то видно, що відступи є лише МІЖ елементами меню. Тобто справа і зліва від останнього і першого їх нема. Тому варто використати margin для одного боку, а для останнього елементу (:last-child) обнулити margin